stable/kong/values.yaml

```diff
@@ -305,8 +305,8 @@ readinessProbe:
 livenessProbe:
   httpGet:
     path: "/status"
-    port: admin
-    scheme: HTTPS
+    port: metrics
```

Comment:
Are we registering `metrics` port somewhere?

Reply:
Yes.
https://github.com/helm/charts/blob/ce04b0bba646957c632435adc5a2212834055fdd/stable/kong/templates/deployment.yaml#L193